src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -182,6 +183,15 @@
       </IncludedFileForRunnerScript>
     </ItemGroup>
     
+    <Copy Condition="'$(TestAppX)' == 'true'"
```

Comment:
We should be avoiding any extra Copying in these targets. The copying should be handled by runtest.cmd/sh now so that we ensure the same thing happens both locally and in helix.